docs/src/examples/components/Ref/Types/RefExampleForwardRef.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
```

Comment:
maybe name like `RefForwardingExample` would be more readable and easier to grasp